pkg/health/server/server_test.go

```diff
@@ -167,55 +167,93 @@ var sampleClustermeshConnectivity = &healthReport{
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="0.5"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="1"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="2"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="4"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="8"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="endpoint",le="+Inf"} 4
+cilium_node_connectivity_latency_seconds_sum{source_cluster="default",source_node_name="kind-worker",type="endpoint"} 0.00858722
+cilium_node_connectivity_latency_seconds_count{source_cluster="default",source_node_name="kind-worker",type="endpoint"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.001"} 1
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.0025"} 2
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.005"} 2
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.01"} 3
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.025"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.05"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.1"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.25"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="0.5"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="1"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="2"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="4"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="8"} 4
+cilium_node_connectivity_latency_seconds_bucket{source_cluster="default",source_node_name="kind-worker",type="node",le="+Inf"} 4
+cilium_node_connectivity_latency_seconds_sum{source_cluster="default",source_node_name="kind-worker",type="node"} 0.030578268000000002
```

Comment:
The unit test for the metric does a string comparison of the metric values, but due to a floating-point issue I believe the strings for the node_connectivity_latency_sum values do not match exactly. For now, I edited the values to include the floating-point error, but I am not sure if that is the 'best practice' to use here. I would appreciate feedback on whether it would be better to use a different comparison approach?